tests/Unit/Rule/RequestAttributeValidationRule/Fixture/UseUndefinedRequestAttribute.php

```diff
@@ -0,0 +1,20 @@
+<?php declare(strict_types = 1);
+
+namespace SaschaEgerer\PhpstanTypo3\Tests\Unit\Rule\RequestAttributeValidationRule\Fixture;
+
+use Psr\Http\Message\ServerRequestInterface;
+
+final class UseUndefinedRequestAttribute
+{
+
+	public function someMethod(): void
+	{
+		$this->getServerRequest()->getAttribute('foo');
+	}
+
+	public function getServerRequest(): ServerRequestInterface
+	{
+		return $GLOBALS['TYPO3_REQUEST'];
```

Comment:
Are we sure we can use this one in a unit test? Should we use a mock instead?

Reply:
I would not create a mock here. It is fine. All tests are green.